payjoin/src/send/error.rs

```diff
@@ -305,7 +305,17 @@ impl ResponseError {
     }
 }
 
-impl std::error::Error for ResponseError {}
+impl std::error::Error for ResponseError {
+    fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
+        use ResponseError::*;
+
+        match self {
+            WellKnown(error) => Some(error),
+            Validation(error) => Some(error),
+            Unrecognized { error_code: _, message: _ } => None,
```

Comment:
two dots to elide the named fields of a variant are shorthand for the same effect

```suggestion
            Unrecognized { .. } => None,
```